services/QuillLMS/app/models/blog_post.rb

```diff
@@ -75,6 +75,7 @@ class BlogPost < ApplicationRecord
   ]
 
   TEACHER_TOPICS = TOPICS.reject { |t| [PRESS_RELEASES, IN_THE_NEWS].include?(t) }
+  PUBLIC_TEACHER_TOPICS = TEACHER_TOPICS - [USING_QUILL_FOR_READING_COMPREHENSION]
```

Comment:
How about moving this to the spec file, so that we're not adding app code for the sake of testing?